### DISCUSSION TITLE
Bottom Drawer with Swipeable edge

### DISCUSSION THREAD

**Discussion by Jamie-VisualiseInfo** [mantinedev/mantine#5734] [Feature requests]
It would be great if there was a bottom drawer with a preview peeking out from the bottom of the screen where you can swipe up on mobile to reveal the full drawer.
![Screenshot 2024-02-07 at 10 58 35](https://github.com/mantinedev/mantine/assets/108797348/4ceb8d83-de25-40e5-984b-1f7fac7a93a9) 
Something similar to this, found in Material UI.
https://mui.com/material-ui/react-drawer/#swipeable-edge

**Comment by Abbosbek-cloud** [comment]
Can we add swipe to close functionality into mantine drawer as mentioned in this discussion @rtivital?
